fv3config/_datastore.py

```diff
@@ -181,6 +181,45 @@ def get_microphysics_name(config):
 def get_field_table_filename(config):
     """Get field_table filename given configuration dictionary
```

Comment:
This function is a little large now, with very low-level code, can it be refactored to call only higher-level functions that describe what they do based on their name, that contain the low-level code? And can it also be refactored to only return at the bottom of the function? It's hard to follow the control-flow.

Reply:
I should add it's not your fault it's like this, it was already getting there before you added this new feature, which pushed it over the top.

Reply:
I refactored things a bit; let me know what you think.